app/views/hosts/_form.html.erb

```diff
@@ -4,7 +4,7 @@
 <%= render "hosts/progress" %>
 
 <% Taxonomy.as_taxonomy @organization , @location do %>
-
+<div id="host_form">
   <%= form_for @host, :url => (@host.new_record? ? hosts_path : host_path(:id => @host.id)), :html => {:data => {:id => @host.try(:id), :type_changed => !!@host.type_changed?, :submit => 'progress_bar'}} do |f| %>
```

Comment:
I think it would be better to add an id to the form instead of wrapping it with another div that may affect layout